app/controllers/shopping_lists_controller.rb

```diff
@@ -23,11 +23,47 @@ def show
     head :not_found
   end
 
+  def update
+    shopping_list = current_user.shopping_lists.find(params[:id])
```

Comment:
Master lists should not be updateable via this route but that work will be done in [this card](https://trello.com/c/tPeoZBxn/3-return-error-for-non-get-methods-on-master-list-resource).